app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  before_save {self.email = email.downcase}
```

Comment:
Can I ask you a question: What is before_save in rails

Reply:
before_save callback the method before save data in table. in this case , It downcases email before save user email to standardize all email